### PR TITLE
Display generated sequences + ODB updates + Reusability.byEq

### PR DESCRIPTION
**Pull request by rpiaggio**
Only future steps are shown for the moment (past visits will be addressed in the future).

No generation input parameters yet, also will be addressed in future PR.

image

### CODE REVIEW

File: common-graphql/src/main/scala/queries/common/ManualSequenceGQL.scala [hunk 1]
```diff
@@ -0,0 +1,269 @@
...
+
+// gql: import lucuma.schemas.decoders._
+// gql: import lucuma.ui.reusability._
+
+object ManualSequenceGQL {
+
+  @GraphQL
+  trait SequenceSteps extends GraphQLOperation[ObservationDB] {
+    val document = """
+      query($programId: ProgramId!) {
```
Comment (cquiroz) [inline common-graphql/src/main/scala/queries/common/ManualSequenceGQL.scala:22]: general question. this queries are validated by the scalafix steps right? Just asking to confirm these are validated at compile time

File: explore/src/main/scala/explore/config/ManualSequenceTables.scala [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+import japgolly.scalajs.react._
+import japgolly.scalajs.react.vdom.html_<^._
+import lucuma.core.model.sequence._
+import lucuma.ui.reusability._
+import react.common._
+import react.semanticui.elements.header.Header
+import react.semanticui.elements.segment.Segment
+
+final case class ManualSequenceTables(config: ManualConfig)
```
Comment (cquiroz) [inline explore/src/main/scala/explore/config/ManualSequenceTables.scala:14]: This looks very similar to `GeneratedSequenceTable` could it be generated?

File: explore/src/main/scala/explore/config/SpectroscopyModesTable.scala [hunk 3]
```diff
@@ -310,32 +310,29 @@ object SpectroscopyModesTable {
         .setSortType(DefaultSortTypes.number)
     ).filter { case c => (c.id.toString) != FPUColumnId.value || fpu.isEmpty }
 
-  protected def rowToConf(row: SpectroscopyModeRow): Option[ScienceConfiguration] =
+  protected def rowToConf(row: SpectroscopyModeRow): Option[ScienceModeBasic] =
```
Comment (cquiroz) [inline explore/src/main/scala/explore/config/SpectroscopyModesTable.scala:313]: Great to see the ITC queries updated

**Review by cquiroz — APPROVED** [review]
Great work, just a few comments

File: common-graphql/src/main/scala/queries/common/ManualSequenceGQL.scala (hunk 1 above)
Comment (rpiaggio) [reply common-graphql/src/main/scala/queries/common/ManualSequenceGQL.scala:22]: Right, they are checked against the schema by the scalafix rule, which is run whenever we compile.

File: explore/src/main/scala/explore/config/ManualSequenceTables.scala (hunk 2 above)
Comment (rpiaggio) [reply explore/src/main/scala/explore/config/ManualSequenceTables.scala:14]: They are very similar indeed. I expect them to diverge in the future. I'll unify them nonetheless in a next PR.

**Pull request merged**